cli/man/splinter-circuit-template-arguments.1.md

```diff
@@ -22,6 +22,9 @@ Circuit templates help simplify the process of creating new circuits with the
 building a circuit from the template. This command lists the arguments that are
 defined in the specified circuit template.
 
+All available templates are located in the default circuit templates directory,
```

Comment:
If there's no way to use a different directory for the circuit templates, remove the word "default" (here and in all other places/man pages that reference this directory).